src/Symfony/Component/Validator/Constraints/FileValidator.php

```diff
@@ -292,6 +307,33 @@ private static function moreDecimalsThan($double, $numberOfDecimals)
         return strlen((string) $double) > strlen(round($double, $numberOfDecimals));
     }
 
+    private static function getMaxFilesize()
```

Comment:
This method is equal to `HttpFoundation\File\UploadedFile::getMaxFilesize()`. It exists here in order to remove the dependency on HttpFoundation for creating the error message.